packages/tocco-ui/src/Menu/Item.js

```diff
@@ -0,0 +1,66 @@
+import PropTypes from 'prop-types'
+import React from 'react'
+import styled from 'styled-components'
+
+const ItemStyles = styled.li`
```

Comment:
I would prefer the name `StyledItem`, because it actually is an item *with* styles.

"ItemStyles" indicates that this object represents just the styles (without the item), which is not the case.

Reply:
i'm on rolis side on this one. BlaStyles is a bit confisung

Reply:
see 70afaf07d953a208b2f9843bda3dc76c17d5d5b2